packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -458,13 +468,22 @@ export default class TabAdvanced extends Component {
                         value={backupNetwork}
                         xoType='network'
                       >
-                        {backupNetwork !== undefined ? <Network id={backupNetwork.id} /> : _('noValue')}
+                        {pool.otherConfig['xo:backupNetwork'] === undefined ? (
+                          _('noValue')
+                        ) : backupNetwork !== undefined ? (
+                          <Network id={backupNetwork.id} />
+                        ) : (
+                          <span className='text-danger'>
+                            {_('updateMissingNetwork', {
+                              networkID: (
+                                <Copiable onClick={e => e.stopPropagation()}>
```

Comment:
I tested, and the problem is that the UUID no longer becomes clickable. This can be confusing.
Finally, keep the previous behavior. We might fix this in a nicer way in another PR, later.